Workbooks/Communication Services/Email_Insights/Email_Insights.workbook

```diff
@@ -0,0 +1,983 @@
+              "subTarget": "stats",
+              "style": "link",
+              "linkIsContextBlade": true
+            },
+            {
+              "id": "7667d622-e3a3-46f9-98b4-ecd7f4deee3d",
+              "cellValue": "tab",
+              "linkTarget": "parameter",
+              "linkLabel": "Email Performance",
+              "subTarget": "perf",
+              "style": "link"
+            }
+          ]
+        },
+        "name": "links - 4"
+      },
+      {
+        "type": 9,
+        "content": {
+          "version": "KqlParameterItem/1.0",
+          "parameters": [
+            {
+              "id": "1a744cde-5e97-45e8-9966-ccef8b6a698d",
+              "version": "KqlParameterItem/1.0",
+              "name": "Time",
+              "type": 4,
+              "isGlobal": true,
+              "typeSettings": {
+                "selectableValues": [
+                  {
+                    "durationMs": 3600000
+                  },
+                  {
+                    "durationMs": 14400000
+                  },
+                  {
+                    "durationMs": 43200000
+                  },
+                  {
+                    "durationMs": 86400000
+                  },
+                  {
+                    "durationMs": 172800000
+                  },
+                  {
+                    "durationMs": 259200000
+                  },
+                  {
+                    "durationMs": 604800000
+                  },
+                  {
+                    "durationMs": 2592000000
+                  },
+                  {
+                    "durationMs": 5184000000
+                  },
+                  {
+                    "durationMs": 7776000000
+                  }
+                ],
+                "allowCustom": true
+              },
+              "timeContext": {
+                "durationMs": 86400000
+              },
+              "value": {
+                "durationMs": 777600000,
+                "endTime": "2023-07-10T22:51:00.000Z"
+              }
+            },
+            {
+              "id": "36134be5-8638-4f33-8fcc-0432d0b9cf63",
+              "version": "KqlParameterItem/1.0",
+              "name": "mailfrom",
+              "label": "Mail From Address",
+              "type": 2,
+              "isRequired": true,
+              "query": "ACSEmailStatusUpdateOperational \r\n| where OperationName == \"DeliveryStatusUpdate\" and isnotempty(DeliveryStatus)and isnotempty(RecipientId) and DeliveryStatus != \"OutForDelivery\" \r\n| summarize by sender=strcat(SenderUsername, '@', SenderDomain)",
+              "typeSettings": {
+                "additionalResourceOptions": [],
+                "showDefault": false
+              },
+              "queryType": 0,
+              "resourceType": "microsoft.communication/communicationservices",
+              "value": "donotreply@15edc21a-35b8-4d3d-92b7-48a4a4dccada.us1.azurecomm.net"
```

Comment:
Can we make this generic rather than a specific guid?

Reply:
Good catch! I will remove this value